### PR TITLE
Actions

### PR DESCRIPTION
**Pull request by jburel**
Replace travis by GHA

cc @sbesson

### CODE REVIEW

**Review by sbesson — COMMENTED** [review]
Looks good. One inline question about the release step

Otherwise, what is the driver for implementing a multi-step workflow as opposed to three independant templated workflows (tox, omero, pypi)?

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      - test_infra
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
+      - name: Set up
+        run: |
+          pip install https://github.com/ome/zeroc-ice-py-manylinux/releases/download/0.1.0/zeroc_ice-3.6.5-cp36-cp36m-manylinux2010_x86_64.whl
```
[the remark this answers is not in the document]
Comment (jburel) [reply .github/workflows/workflow.yml:67]: I copied it from travis when I was testing something and it should be removed

**Comment by jburel** [comment]
I had initial separate files then I opted so synch with https://github.com/ome/omero-py/blob/master/.github/workflows/workflow.yml
Happy to revert, probably time to synch repo and declare strategy in template repo

**Comment by sbesson** [comment]
> I had initially multiples files then I opted so synch with https://github.com/ome/omero-py/blob/master/.github/workflows/workflow.yml
>Happy to revert, probably time to synch repo and declare strategy in template repo

Understood. The original decision around single vs multiple workflows happened in https://github.com/ome/omero-web/pull/240 and all the subsequent migration for the Python repositories has been following the principles of this PR. `omero.yml` and `publish_pypi.yml` have already been migrated upstream as workflow templates. Adding `tox.yml` as a template workflow  and updating `omero-py` to follow the multiple workflows layout is on my todo list.

**Comment by jburel** [comment]
ok so I will split this one.

**Review by sbesson — APPROVED** [review]

File: .github/workflows/tox.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        run: python -mpip install --upgrade wheel pytest tox
+      - name: Get tox target
+        id: toxtarget
+        run: |
+          py=$(echo ${{ matrix.python-version }} | tr -d .)
+          echo "::set-output name=py::$py"
+      - name: Run tests
+        run: tox -e py${{ steps.toxtarget.outputs.py }}
+
+  test_infra:
```
Comment (sbesson) [inline .github/workflows/tox.yml:36]: Sorry I missed that these steps hadn't been removed from this renamed files. Should the workflow be simplified to only run `tox`?

**Pull request merged**